src/webgpu/shader/execution/expression/unary/unary.ts

```diff
@@ -1,6 +1,6 @@
 import { ExpressionBuilder } from '../expression.js';
 
-/* @returns an ExpressionBuilder that evaluates the prefix unary operation */
+/* @returns an ExpressionBuilder that evaluates a prefix unary operation */
 export function unary(op: string): ExpressionBuilder {
-  return value => `${op}${value}`;
+  return value => `${op}(${value})`;
```

Comment:
To handle booleans being ints in storage the framework is going to insert != 0u after value, so using () to avoid the shader being `!param != 0u` instead of `!(param != 0u)`